hisiodetbt.src

```diff
@@ -0,0 +1,182 @@
+;  highsiodet.src - check for highspeed drives
+;
+;  Copyright (c) by Matthias Reichl <hias@horus.com>
```

Comment:
Please add a copyright entry with your name and date (eg 2019 or 2019-2020)

Reply:
OK